.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,24 @@
+name: update-breaking-change-status
+on:
+  pull_request:
+    types:
+      - labeled
+jobs:
+  update-breaking-change-status:
+    if: github.event.label.id == '4598495472' # The unique identifier for the override-breaking-change label.. used in place of name so the name can be modified without breaking usage
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
```

Comment:
just to double-check, does it need these permissions if it's only updating the `statuses` endpoint?

Reply:
Should be doable with read only